src/components/main/ItemScreens/ItemScreenLayout.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect } from 'react';
+import { Outlet } from 'react-router';
+
+import { Loader, useShortenURLParams } from '@graasp/ui';
+
+import { ITEM_ID_PARAMS } from '@/config/paths';
+
+import { PERMISSIONS_EDITION_ALLOWED } from '../../../config/constants';
+import { hooks } from '../../../config/queryClient';
+import { getHighestPermissionForMemberFromMemberships } from '../../../utils/membership';
+import ErrorAlert from '../../common/ErrorAlert';
+import { useCurrentUserContext } from '../../context/CurrentUserContext';
+import { useLayoutContext } from '../../context/LayoutContext';
+import { UppyContextProvider } from '../../file/UppyContext';
+import Main from '../Main';
+import WrappedAuthItemScreen from './WrappedItemScreen';
+
+const { useItem, useItemMemberships } = hooks;
+
+const ItemScreen = (): JSX.Element => {
+  const itemId = useShortenURLParams(ITEM_ID_PARAMS);
+
+  const { data: item, isError, isLoading } = useItem(itemId);
+  const { setEditingItemId } = useLayoutContext();
+  const { data: currentMember } = useCurrentUserContext();
+  const {
+    data: memberships,
+    isLoading: isLoadingItemMemberships,
+    isError: isErrorItemMemberships,
+  } = useItemMemberships(itemId);
+
+  useEffect(() => {
+    setEditingItemId(null);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [itemId]);
+
+  if (isLoading || isLoadingItemMemberships) {
+    return <Loader />;
+  }
+
+  if (!itemId || !item || isError || isErrorItemMemberships) {
+    return <ErrorAlert />;
+  }
```

Comment:
Basile told me it's better to have `isLoading` and errors at the end of the component. So instead you would have
```
if(item) {
  const itemMembership = ...
}

if(isLoading) { ... }

return <ErrorAlert />;
```